journal-week-2.md

```diff
@@ -1,14 +1,14 @@
 
 ###Proof of Life
 
-We made it through the first week, and we are still alive. the second week was like a recovery period for us after the shock.  Most of us started to understand how this course will look like. That we will be bombarded with assignment especially on weekends, and from there we will learn to solve problems though research, team work and alot of educated guesses. 
+We made it through the first week, and we are still alive. the second week was like a recovery period for us after the shock. Most of us started to understand how this course will look like - that we will be bombarded with assignment especially on weekends, and from there we will learn to solve problems through research, team work and alot of educated guesses.
 
-Last week, we learned a lot:
+Last week, we learned a lot of new things:
 
-**More of Githubbing** - more exposure in terminal, solving conflicts. Using source tree as alternative.
+More of Githubbing - more exposure in CLI, solving conflicts. Using source tree as alternative. I thought I was comfortable using this VCS, but it seems everyday we encounter different conflicts and because of that we continue learning more about the system as we work with it.
 
-**Understanding JS** - continued improving our skills in JS by applying it to more practical ways. i.e. linking it with html.
+Understanding JS - continued improving our skills in JS by applying it to more practical ways. i.e. linking it with html and in a a way with css.
 
-**More testing of JS via node and browser** - accessing html, css and JS files in browser
+More testing of JS via node and browser - accessing html, css and JS files in browser particularly chrome dev tools.
 
-**DOM** - we had a deep introduction of the DOM, and had a great undertsnding of it now. We managed to control the DOM behaviours using vanilla javascript. I have learned a lot in this particular tasks by trying my best to understand pseudocoding and impeement it with javascript. I also learned here to work with other students and work as a team. 
+DOM - we had a deep introduction of the DOM, and had a great undertsnding of it now. We managed to control the DOM behaviours using vanilla javascript. I have learned a lot in this particular tasks by trying my best to understand pseudocoding and impeement it with javascript. I also learned last week to work with other students and work as a team.
```

Comment:
we had a deep introduction of the DOM, and HAVE a great UNDERSTANDING of it now....

Behaviors is spelled without a "u"....

....particular TASK...not "tasks"...

..."impeement" should be "IMPLEMENT"